docs/pages/guides/quick-start.md

```diff
@@ -43,6 +53,21 @@ This guide will explain basic LiveCompositor setup.
 ### Register input stream `input_1`.
 
 <Tabs queryString="lang">
+  <TabItem value="react" label="React">
+    ```tsx
+    await compositor.registerInput("input_1", {
+      type: "rtp_stream",
+      transportProtocol: "tcp_server",
+      port: 9001,
+      video: {
+        decoder: "ffmpeg_h264"
+      }
+    })
+    ```
```

Comment:
I think it would be great for react guide, if we could have full working code expendable (with all imports / previously defined code etc.), such that users could copy-paste it.